src/applications/accreditation/21a/config/form.js

```diff
@@ -0,0 +1,88 @@
+// import fullSchema from 'vets-json-schema/dist/21A-schema.json';
+
+import {
+  dateOfBirthSchema,
+  dateOfBirthUI,
+  fullNameSchema,
+  fullNameUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import FormFooter from 'platform/forms/components/FormFooter';
+import { VA_FORM_IDS } from 'platform/forms/constants';
+
+import GetFormHelp from '../components/GetFormHelp';
+import ConfirmationPage from '../containers/ConfirmationPage';
+import IntroductionPage from '../containers/IntroductionPage';
+import manifest from '../manifest.json';
+
+// const { } = fullSchema.properties;
+// const { } = fullSchema.definitions;
+
+const formConfig = {
+  formId: VA_FORM_IDS.FORM_21A,
+  version: 0,
+  rootUrl: manifest.rootUrl,
+  urlPrefix: '/',
+  // submitUrl: '/v0/api',
```

Comment:
Should we indicate why this is commented out for now (maybe a TODO with an issue number)?

Reply:
> Definitely work to create issues and solutions for each of these areas in forthcoming work. Since LGTM I'll merge
A // TODO comment would be useful to future devs in these cases!